src/components/ContentListTable/components/PackageCount.tsx

```diff
@@ -32,6 +32,14 @@ const PackageCount = ({ rowData }: Props) => {
     );
   }
 
+  if (rowData.status === 'Invalid') {
+    return (
+      <Tooltip isContentLeftAligned content='Repository is invalid please review it'>
```

Comment:
```suggestion
      <Tooltip isContentLeftAligned content='Repository is invalid.'>
```
Better punctuation would be "Repository is invalid, please review it.' but style guides say not to use "please" so its best just to make the tool tip short and to the point.